components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/dao/OrganizationManagementDAO.java

```diff
@@ -385,6 +386,9 @@ String getAnAncestorOrganizationIdInGivenDepth(String organizationId, int depth)
      * @param rootOrganization The root organization to be added.
      * @throws OrganizationManagementServerException The server exception thrown when adding a root organization.
      */
-    void addRootOrganization(Organization rootOrganization) throws OrganizationManagementServerException;
+    default void addRootOrganization(Organization rootOrganization) throws OrganizationManagementServerException {
+
+        throw new NotImplementedException();
```

Comment:
no need to throw not implemented exception in the void method.
just add a debug log

Reply:
As this is DAO level interface which is not supposed to be public, removed the default method. But Organization manager interface method kept the default by throwing not implemented exception as it should return an Organization object. 
- https://github.com/wso2/identity-organization-management-core/pull/89/commits/2b9136af99f90c812b82919da8e7f67b70df2336